### PR TITLE
scan-return-of-results: Return Spanish and zh-Hant results

### PR DESCRIPTION
**Pull request by kairstenfay**
Return Spanish SCAN IRB results, exporting these data from REDCap and
generating Spanish PDFs. Bump the Docker image version used to create
these PDFs so that it uses the latest lab-result-reports template.

Depends on https://github.com/seattleflu/lab-result-reports/pull/16 and https://github.com/seattleflu/id3c-customizations/pull/111

---------------------------

This PR now updated to include recently finalized zh-Hant changes to reduce code churn

### CODE REVIEW

**Comment by kairstenfay** [comment]
[SCAN IRB translations - Spanish](https://trello.com/c/yQR01au0/629-scan-irb-translations-spanish)

File: crontabs/id3c-production [hunk 1]
```diff
@@ -89,6 +89,7 @@ ENVD=/opt/backoffice/id3c-production/env.d
 # Ingest each SCAN Research REDCap project every 30 min (with the
 # first starting 5 minutes after the SFS/SCAN Surveillance ingests above.)
 5,35 * * * * ubuntu flock -F /home/ubuntu/cache.pickle envdir $ENVD/deidentification envdir $ENVD/smartystreets envdir $ENVD/redcap-scan-irb-en pipenv run id3c etl redcap-det scan-irb-en --commit
+7,37 * * * * ubuntu flock -F /home/ubuntu/cache.pickle envdir $ENVD/deidentification envdir $ENVD/smartystreets envdir $ENVD/redcap-scan-irb-es pipenv run id3c etl redcap-det scan-irb-es --commit
```
Comment (joverlee521) [inline crontabs/id3c-production:92]: Don't forget to add `redcap-scan-irb-es` to `env.d`

Comment (kairstenfay) [reply crontabs/id3c-production:92]: Oh, right. I frequently forget that it needs to be tracked in this repo, not just on the backoffice server.

Comment (kairstenfay) [reply crontabs/id3c-production:92]: I fixed this up into the latest commit.

**Comment by kairstenfay** [comment]
The traditional Chinese lab results report is now finalized, so include zh-Hant changes in this PR.

**Comment by kairstenfay** [comment]
[SCAN IRB translations - Traditional Chinese](https://trello.com/c/DRgbhIUu/637-scan-irb-translations-traditional-chinese)

**Pull request merged**